flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/ExecutionContext.java

```diff
@@ -119,7 +123,6 @@
 	private final Map<String, TableSink<?>> tableSinks;
 	private final Map<String, UserDefinedFunction> functions;
 	private final Configuration flinkConfig;
-	private final Configuration executorConfig;
 	private final ClusterClientFactory<ClusterID> clusterClientFactory;
 	private final ExecutionConfigAccessor executionParameters;
```

Comment:
This does not seem to be used anymore.

Reply:
It seems so. If so, we can remove `createExecutionParameterProvider` also.

Reply:
👌